services/app/src/components/views/Settings.tsx

```diff
@@ -46,56 +50,57 @@ const timerText: { [v: string]: any } = [
 export const timerValues: Array<number|null> = [null, 30, 15, 10, 6];
 
 const Settings = (props: Props): JSX.Element => {
-  const difficultyIdx = difficultyValues.indexOf(props.difficulty);
-  const fontSizeIdx = fontSizeValues.indexOf(props.fontSize);
-  const timerIdx = props.timerSeconds ? timerValues.indexOf(props.timerSeconds) : 0;
-
+  const difficultyIdx = difficultyValues.indexOf(props.settings.difficulty);
+  const fontSizeIdx = fontSizeValues.indexOf(props.settings.fontSize);
+  const timerIdx = props.settings.timerSeconds ? timerValues.indexOf(props.settings.timerSeconds) : 0;
+  const adventurers = numAdventurers(props.settings, props.multiplayer);
   return (
     <Card title="Settings">
       <Button className="primary large" onClick={() => props.onExpansionSelect()}>Choose game / expansion</Button>
-      <p className="expansionLabel">Currently playing: <strong>Expedition {props.contentSets.horror ? <span> + Horror</span> : null}{props.contentSets.future ? <span> + Future</span> : null}</strong></p>
+      <p className="expansionLabel">Currently playing: <strong>Expedition {props.settings.contentSets.horror ? <span> + Horror</span> : null}{props.settings.contentSets.future ? <span> + Future</span> : null}</strong></p>
 
-      <Picker label="Adventurers" value={props.numLocalPlayers} onDelta={(i: number) => props.onPlayerDelta(props.numLocalPlayers, i)}>
-        {(props.numLocalPlayers > 1) ? 'The number of players.' : <div><strong>Solo play:</strong> Play as two adventurers with double the combat timer.</div>}
+      <Picker id="playerCount" label="Adventurers" value={props.settings.numLocalPlayers} onDelta={(i: number) => props.onPlayerDelta(props.settings.numLocalPlayers, i)}>
+        {(adventurers > 1) ? 'The number of players.' : <div><strong>Solo play:</strong> Play as two adventurers with double the combat timer.</div>}
+        {props.multiplayer && <div>({adventurers} across all clients)</div>}
```

Comment:
Instead of all "clients" would it make sense to refer to them as "devices"? Since clients and devices mean the same thing / have a 1:1 map, and most folks are more familiar with the term / concept of devices?

Reply:
Renamed to devices.